ged2dot.py

```diff
@@ -777,64 +780,70 @@ def load(self) -> None:
+                    if self.family:
+                        self.model.families.append(self.family)
+                        self.family = None
+
+                    if rest.startswith("@") and rest.endswith("INDI"):
+                        id_string = rest[1:-6]
+                        if id_string not in self.model.config.indiBlacklist:
+                            self.indi = Individual(self.model)
+                            self.indi.id = rest[1:-6]
+                    elif rest.startswith("@") and rest.endswith("FAM"):
+                        self.family = Family(self.model)
+                        self.family.id = rest[1:-5]
+
+                elif level == 1:
                     if self.inBirt:
-                        self.indi.setBirt(year)
+                        self.inBirt = False
                     elif self.inDeat:
-                        self.indi.deat = year
-
+                        self.inDeat = False
+
+                    if rest.startswith("SEX") and self.indi:
+                        self.indi.sex = rest.split(' ')[1]
+                    elif rest.startswith("NAME") and self.indi:
+                        rest = rest[5:]
+                        tokens = rest.split('/')
+                        self.indi.forename = tokens[0].strip()
+                        if len(tokens) > 1:
+                            self.indi.surname = tokens[1].strip()
+                    elif rest.startswith("FAMC") and self.indi:
+                        # Child in multiple families? That's crazy...
+                        if not self.indi.famc:
+                            self.indi.famc = rest[6:-1]
+                    elif rest.startswith("FAMS") and self.indi:
+                        self.indi.fams = rest[6:-1]
+                    elif rest.startswith("BIRT"):
+                        self.inBirt = True
+                    elif rest.startswith("DEAT"):
+                        self.inDeat = True
+                    elif rest.startswith("HUSB") and self.family:
+                        self.family.husb = rest[6:-1]
+                    elif rest.startswith("WIFE") and self.family:
+                        self.family.wife = rest[6:-1]
+                    elif rest.startswith("CHIL") and self.family:
+                        id_string = rest[6:-1]
+                        if id_string not in self.model.config.indiBlacklist:
+                            self.family.chil.append(rest[6:-1])
+
+                elif level == 2:
+                    if rest.startswith("DATE") and self.indi:
+                        year = rest.split(' ')[-1]
+                        if self.inBirt:
+                            self.indi.setBirt(year)
+                        elif self.inDeat:
+                            self.indi.deat = year
+
+            except Exception as e:
+                print("Encountered parsing error in .ged: " + str(e))
+                print("line (%d): %s" % (linecount, line))
+                sys.exit(1)
```

Comment:
Nitpick: surrounding code always uses two empty lines around function / class definitions, keep that here as well.